modules/role/manifests/swift.pp

```diff
@@ -21,7 +21,7 @@
         include role::memcached
 
         $firewall_rules_mediawiki_str = join(
-            query_facts('Class[Role::Mediawiki]', ['ipaddress', 'ipaddress6'])
```

Comment:
Can remove this and just use firewall_rules_swift_str

Reply:
Done